framework_tvb/tvb/interfaces/rest/server/resources/project/project_resource.py

```diff
@@ -33,12 +33,14 @@
 from tvb.interfaces.rest.commons import Strings
 from tvb.interfaces.rest.commons.dtos import OperationDto, DataTypeDto
 from tvb.interfaces.rest.commons.exceptions import InvalidIdentifierException, InvalidInputException
+from tvb.interfaces.rest.server.decorators.rest_decorators import rest_jsonify, token_required
 from tvb.interfaces.rest.server.resources.rest_resource import RestResource
 
 INVALID_PROJECT_GID_MESSAGE = 'No project found for GID: %s'
 
 
 class GetDataInProjectResource(RestResource):
+    method_decorators = [rest_jsonify, token_required]
```

Comment:
How about having the following inheritance: RestResource <-- SecuredRestResource <--- majority of other *Resources ?